src/lib.rs

```diff
@@ -97,6 +98,8 @@ pub trait HasContext {
 
     fn supports_debug(&self) -> bool;
 
+    fn get_version(&self) -> &Version;
```

Comment:
What would you think about shortening this to `version` instead of having the `get_` prefix? Mostly thinking about https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter

Reply:
Excellent suggestion! I have rebased and changed to `Context::version`.